src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5553,8 +5618,11 @@ public override BoundNode VisitDelegateCreationExpression(BoundDelegateCreationE
                     break;
                 case BoundLambda lambda:
                     {
-                        VisitLambda(lambda, delegateType, Diagnostics);
+                        var lambdaSymbol = lambda.CreateLambdaSymbol(delegateType, GetUpdatedContainingSymbol(lambda.Symbol));
```

Comment:
>var lambdaSymbol = lambda.CreateLambdaSymbol(delegateType, GetUpdatedContainingSymbol(lambda.Symbol)); [](start = 24, length = 102)

Same comments as for VisitConversion

Reply:
No longer applies as this code isn't changed.

---
In reply to: [327919573](https://github.com/dotnet/roslyn/pull/38769#discussion_r327919573) [](ancestors = 327919573)